src/cli/db.ts

```diff
@@ -0,0 +1,79 @@
+import { flags } from '@oclif/command'
+import { OutputFlags } from '@oclif/parser'
+
+import { loggingFactory } from '../logger'
+import BaseCommand from '../utils'
+import DbMigration from '../../migrations'
+
+const logger = loggingFactory('cli:db')
+
+export default class DbCommand extends BaseCommand {
+  static hidden: boolean;
+  static flags = {
+    ...BaseCommand.flags,
+    up: flags.boolean({
+      char: 'm',
+      description: 'Migrate DB'
+    }),
+    down: flags.boolean({
+      char: 'u',
+      description: 'Undo db migration'
+    }),
+    from: flags.string({
```

Comment:
Hmm this should not be needed right? This information is stored in the Umzug's table right?

Reply:
It's an optional. You may specify the point to/from you are want to migrate/undo (it just filters the files)

Reply:
Hmm I don't think that it actually make sense. If you run the migrations from different starting point then the actual one you are making the DB inconsistent because you are either skipping migrations or reapplying some migrations twice...